+toml/private/bracketarray.m

```diff
@@ -0,0 +1,83 @@
+function out = bracketarray(in)
+  % BRACKETARRAY TOML representation of a MATLAB numerical array, in the style of a numpy array
+  %
+  %   BRACKETARRAY(array) returns the TOML/numpy-like representation of `array' with nested brackets
+  %   [1, 2; 3, 4] becomes [[1,2],[3,4]]
+
+  %   From: https://stackoverflow.com/questions/57438523/in-matlab-how-can-i-write-out-a-multidimensional-array-as-a-string-that-looks-li/57445408#57445408
+  %   By: matlabbit
+
+  out = permute(in, [2, 1, 3:ndims(in)]);
+  out = string(out);
+
+  dimsToCat = ndims(out);
+  if iscolumn(out)
+    dimsToCat = dimsToCat - 1;
+  end
+
+  for iDim = 1:dimsToCat
+    out = "[" + join(out, ",", iDim) + "]" ;
+  end
+  out = char(out);
+end
+
+%% Test Suite
```

Comment:
can you add these to the test suite? should be relatively simple if you parameterize the tests.

Reply:
Well, the formal unit test code is somewhat new to me (I can understand what's going on in the tests you've written but I've never made new tests before). So yea I'll give it a shot and try to add these. When you say it should be simple if I parameterize them, what exactly do you mean? 

And thank you for getting back to this, I'm sorry if I was coming off as pushy before, I didn't intend to.

Reply:
no worries, github did not notify me at all about this PR or the issues opened on the repo. 

[here is an intro to parameterized testing in matlab](https://www.mathworks.com/help/matlab/matlab_prog/create-basic-parameterized-test.html) - basically you define a `properties` block with a few variables to iterate over, then you can use those as arguments to the test `methods`.

it's used [here in the `TestTomlDecode` test](https://github.com/g-s-k/matlab-toml/blob/master/%2Btoml/%2Btesting/%40TestTomlDecode/TestTomlDecode.m#L3) if you want to see it in practice.

Reply:
Thank you! I'll look into it. Will github notify you of any changes or future PR's? If not, how should I best contact you?

Reply:
what you would likely want to do is define a new test class (e.g. `+toml/+testing/@TestBracketArray`) like this (warning: pseudocode incoming)

```matlab
classdef TestBracketArray < matlab.unittest.TestCase
  properties (TestParameter)
    input = { %{ list of input arrays %} };
    output = { %{ list of outputs %} };
  end

  methods (Test, ParameterCombination = 'sequential')
    function testValidInputs(testCase, input, output)
      testCase.verifyEqual(bracketArray(input), output);
    end
  end
end
```

Reply:
i believe if you "@" me i will be notified, that works fine for me.